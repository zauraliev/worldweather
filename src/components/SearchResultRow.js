import React from "react";

const SearchResultRow = props => (
  <div className="row">
    <div className="col-md-3 text-lg-left">
      <label className="font-weight-bold list-header mr-1">{props.label}</label>
    </div>
    <div className="col-md-6 text-lg-left">
      <span>{props.value}</span>
    </div>
  </div>
);

export default SearchResultRow;
