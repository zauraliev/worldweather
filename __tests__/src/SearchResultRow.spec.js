import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import SearchResultRow from "../../src/components/SearchResultRow";

describe("SearchResultRow", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<SearchResultRow />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
