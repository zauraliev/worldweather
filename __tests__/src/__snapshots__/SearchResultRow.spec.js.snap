// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResultRow renders correctly 1`] = `
<div
  className="row"
>
  <div
    className="col-md-3 text-lg-left"
  >
    <label
      className="font-weight-bold list-header mr-1"
    />
  </div>
  <div
    className="col-md-6 text-lg-left"
  >
    <span />
  </div>
</div>
`;
